src/tactic/fin_cases.lean

```diff
@@ -60,7 +66,11 @@ meta def fin_cases_at_aux : Π (with_list : list expr) (e : expr), tactic unit
   | _ := failed
   end)
 
-
+/--
+`fin_cases_at with_list e` performs case analysis on `e : α`, where `α` is a fintype.
+The optional list of expressions `with_list` provides descriptions for the cases of `e`,
+for example, to display nats as `n.succ` instead of `n+1`.
```

Comment:
Could you explain what the values in `with_list` should look like? AFAICT, each should be defeq to an element of the `fintype`, in the order of the fintype, e.g. `fin_cases (n : fin 2) with [0, 0.succ]`.